lib/jekyll/scholar/utilities.rb

```diff
@@ -30,6 +30,10 @@ def optparse(arguments)
         return if arguments.nil? || arguments.empty?
 
         parser = OptionParser.new do |opts|
+          opts.on('-b', '--bib_count') do |bib_count|
```

Comment:
Using the `bibliography_count` tag, we don't really need the option anymore, do we?